bindings/nodejs/examples/exchange/1-generate-mnemonic.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * This example generates a new random mnemonic
+ */
+
+const { AccountManager, CoinType } = require('@iota/wallet');
+
+async function run() {
+    try {
+        const manager = new AccountManager({
+            storagePath: './mnemonic-generation',
+            clientOptions: {
+                nodes: ['http://localhost:14265'],
+            },
+            coinType: CoinType.Shimmer,
+            secretManager: "Placeholder",
```

Comment:
What is `"Placeholder"`?

Reply:
It's a SecretManager type that can be used if you don't want to create a real SecretManager with a seed (which we don't have in this example before)

Reply:
Added a comment